applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -151,10 +151,8 @@ def AddVariables(self):
             self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION_MOMENT)
             self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.POINT_MOMENT)
         # Add variables that the user defined in the ProjectParameters
-        for i in range(self.settings["auxiliary_variables_list"].size()):
-            variable_name = self.settings["auxiliary_variables_list"][i].GetString()
-            variable = KratosMultiphysics.KratosGlobals.GetVariable(variable_name)
-            self.main_model_part.AddNodalSolutionStepVariable(variable)
+        from KratosMultiphysics.auxiliary_solver_utilities import AddVariables
```

Comment:
please ALWAYS put imports at the top of the file (unless it is not possible)
We recently discovered that this can cause problems with recursive imports